frontend/src/layout/header/UserBox.tsx

```diff
@@ -33,71 +33,135 @@ export const UserBox: React.FC = () => {
+                ? <FiAlertTriangle css={iconCss} />
+                : user === "none"
+                    ? <LoggedOut />
+                    : <LoggedIn {...{ t, user }} />
+        }
+    </>;
 };
 
 
 /** User-related UI in header when the user is NOT logged in. */
 const LoggedOut: React.FC = () => {
     const { t } = useTranslation();
 
-    return (
-        <div css={{ display: "flex", padding: "8px 0" }}>
-            <Link
-                to={CONFIG.auth.loginLink ?? LOGIN_PATH}
-                onClick={() => {
-                    // Store a redirect link in session storage.
-                    window.sessionStorage.setItem(REDIRECT_STORAGE_KEY, window.location.href);
-                }}
-                htmlLink={!!CONFIG.auth.loginLink}
-                css={{
-                    alignSelf: "center",
-                    borderRadius: 10,
-                    cursor: "pointer",
-                    padding: "5px 14px",
-                    marginRight: 8,
-                    display: "flex",
-                    gap: 8,
-                    alignItems: "center",
-                    backgroundColor: "var(--nav-color)",
-                    color: "var(--nav-color-bw-contrast)",
-                    "&:hover, &:focus": {
+    return <>
+        <Link
+            to={CONFIG.auth.loginLink ?? LOGIN_PATH}
+            onClick={() => {
+                // Store a redirect link in session storage.
+                window.sessionStorage.setItem(REDIRECT_STORAGE_KEY, window.location.href);
+            }}
+            htmlLink={!!CONFIG.auth.loginLink}
+            css={{
+                outline: "transparent",
+                ":hover, :focus": {
+                    "> :first-child": {
                         backgroundColor: "var(--nav-color-dark)",
                         color: "var(--nav-color-bw-contrast)",
                     },
-                    "&:focus-visible": {
-                        outline: "none",
-                        boxShadow: "0 0 0 2px black",
-                    },
-                    [`@media (max-width: ${BREAKPOINT_MEDIUM}px)`]: {
+                },
+                ":hover > div": {
+                    opacity: 1,
+                    outline: "2px solid var(--grey80)",
+                },
+                ":focus > div": {
+                    opacity: 1,
+                    outline: "2px solid var(--accent-color)",
+                },
+            }}
+        >
+            <div css={{
+                color: "var(--nav-color-bw-contrast)",
+                display: "flex",
+                alignItems: "center",
+                gap: 8,
+                borderRadius: 8,
+                padding: "7px 14px",
+                backgroundColor: "var(--nav-color)",
+                outlineOffset: 1,
+                [`@media (max-width: ${BREAKPOINT_MEDIUM}px)`]: {
+                    display: "none",
+                },
+            }}>
+                <FiLogOut size={"20px"} />
```

Comment:
```suggestion
                <FiLogOut size={20} />
```

That should work as well?